terraform/install.sh

```diff
@@ -150,15 +215,15 @@ installTerraform
 #export GOOGLE_APPLICATION_CREDENTIALS=""
 #gcloud auth application-default login
 
-# Ensure no google.com accounts early - they are not supported!
 acct=$(gcloud info --format="value(config.account)")
 if [[ $acct == *"google.com"* ]];
 then
-  log "Google.com accounts are currently not supported by Stackdriver Sandbox.";
-  exit;
+  folder_id="262044416022"            
```

Comment:
add a comment explaining that this is the /experimental-gke folder

Reply:
Sure.